google/cloud/security/common/data_access/violation_dao.py

```diff
@@ -105,5 +112,10 @@ def _format_violation(violation, resource_name):
     Returns:
         Formatted violations
     """
+
     formatted_output = vm.VIOLATION_MAP[resource_name](violation)
     return formatted_output
+
+Violation = namedtuple('Violation',
+                       ['resource_type', 'resource_id', 'rule_name',
```

Comment:
Can you define this list separately (same file is fine) and make it a frozen list?